### PR TITLE
Upgrade to Qt 5.13.1 on Linux

### PR DESCRIPTION
**Pull request by omichel**
This PR will also help to progress with the snap package and hopefully fix the dependency problem of QtWebEngine on `/sys/devices/pci0000:00/0000:00:17.0/resource` as explained here: https://forum.snapcraft.io/t/qtwebengine-blocked-by-apparmor/13312

### CODE REVIEW

**Comment by omichel** [comment]
👍

**Comment by DavidMansolino** [comment]
(I launched the compilation+test suite on Travis)

**Comment by omichel** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:eyes:

File: dependencies/Makefile.linux [hunk 1]
```diff
@@ -31,9 +32,9 @@ cleanse: $(PACKAGES_CLEAN)
 qt-clean:
 	rm -rf $(WEBOTS_DEPENDENCY_PATH)/$(QT_PACKAGE) $(WEBOTS_HOME)/$(QT_PACKAGE)* $(WEBOTS_HOME)/lib/libQt* $(WEBOTS_HOME)/lib/libicu* $(WEBOTS_HOME)/lib/qt $(WEBOTS_HOME)/include/qt $(WEBOTS_HOME)/bin/qt/lrelease $(WEBOTS_HOME)/bin/qt/lupdate $(WEBOTS_HOME)/bin/qt/moc $(WEBOTS_HOME)/resources/web/local/qwebchannel.js
```
Comment (DavidMansolino) [inline dependencies/Makefile.linux]:
```suggestion
	rm -rf $(WEBOTS_DEPENDENCY_PATH)/$webots-qt-*-linux64-release.tar.bz2 $(WEBOTS_HOME)/$(QT_PACKAGE)* $(WEBOTS_HOME)/lib/libQt* $(WEBOTS_HOME)/lib/libicu* $(WEBOTS_HOME)/lib/qt $(WEBOTS_HOME)/include/qt $(WEBOTS_HOME)/bin/qt/lrelease $(WEBOTS_HOME)/bin/qt/lupdate $(WEBOTS_HOME)/bin/qt/moc $(WEBOTS_HOME)/resources/web/local/qwebchannel.js
```
```

Comment (DavidMansolino) [reply dependencies/Makefile.linux]: It seems good to remove all version of the archive (simpler for switching branches).

**Comment by DavidMansolino** [comment]
I have a few issues with SSL issues:
```
david@cyberbotics-david:~/webots$ ./webots  
Warning: QSslSocket: cannot resolve OPENSSL_init_ssl
Warning: QSslSocket: cannot resolve OPENSSL_init_crypto
Warning: QSslSocket: cannot resolve ASN1_STRING_get0_data
Warning: QSslSocket: cannot resolve EVP_CIPHER_CTX_reset
Warning: QSslSocket: cannot resolve RSA_bits
Warning: QSslSocket: cannot resolve OPENSSL_sk_new_null
Warning: QSslSocket: cannot resolve OPENSSL_sk_push
Warning: QSslSocket: cannot resolve OPENSSL_sk_free
Warning: QSslSocket: cannot resolve OPENSSL_sk_num
Warning: QSslSocket: cannot resolve OPENSSL_sk_pop_free
Warning: QSslSocket: cannot resolve OPENSSL_sk_value
Warning: QSslSocket: cannot resolve DH_get0_pqg
Warning: QSslSocket: cannot resolve SSL_CTX_set_options
Warning: QSslSocket: cannot resolve SSL_CTX_set_ciphersuites
Warning: QSslSocket: cannot resolve SSL_set_psk_use_session_callback
Warning: QSslSocket: cannot resolve SSL_get_client_random
Warning: QSslSocket: cannot resolve SSL_SESSION_get_master_key
Warning: QSslSocket: cannot resolve SSL_session_reused
Warning: QSslSocket: cannot resolve SSL_set_options
Warning: QSslSocket: cannot resolve TLS_method
Warning: QSslSocket: cannot resolve TLS_client_method
Warning: QSslSocket: cannot resolve TLS_server_method
Warning: QSslSocket: cannot resolve X509_up_ref
Warning: QSslSocket: cannot resolve X509_STORE_CTX_get0_chain
Warning: QSslSocket: cannot resolve X509_getm_notBefore
Warning: QSslSocket: cannot resolve X509_getm_notAfter
Warning: QSslSocket: cannot resolve X509_get_version
Warning: QSslSocket: cannot resolve X509_STORE_set_ex_data
Warning: QSslSocket: cannot resolve X509_STORE_get_ex_data
Warning: QSslSocket: cannot resolve OpenSSL_version_num
Warning: QSslSocket: cannot resolve OpenSSL_version
Warning: Incompatible version of OpenSSL
Warning: QSslSocket::connectToHostEncrypted: TLS initialization failed
```

**Comment by omichel** [comment]
I fixed the behavior on Ubuntu 16.04, it now uses libssl1.1 from Ubuntu 18.04.

**Comment by omichel** [comment]
👍

**Comment by DavidMansolino** [comment]
:eyes:

**Comment by DavidMansolino** [comment]
I have fixed some minor issues, recompiled openssl-1.1 on Ubuntu 16.04 and updated the ftp package and now everything is working fine (without warnings) on my environment, I am starting a travis test.

**Pull request merged**